docs/components/arm/_index.md

```diff
@@ -88,9 +88,13 @@ Arm drivers are also paired, in the RDK, with JSON files that describe the kinem
 
 ## Troubleshooting
 
-You can find additional assistance in the [Troubleshooting section](/appendix/troubleshooting/).
+If your arm is not working as expected, follow these steps:
```

Comment:
nice much better. But, [opt] shouldn't this use the include as well? I realize it says "your arm model's" instead of "this model's" but couldn't they both say "Review the arm model's attribute documentation to ensure..."

Same opt comment for all APIs.